src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -70,3 +66,19 @@ class ZuoraService(config: ZuoraConfig, client: FutureHttpClient, baseUrl: Optio
     decode[List[ZuoraErrorResponse]](responseBody).map(_.head)
 
 }
+
+object GetAccountForIdentity {
+
+  case class DomainAccount(accountNumber: String, createdDate: LocalDateTime)
+
+  object DomainAccount {
+
+    def fromAccountRecord(accountRecord: AccountRecord): DomainAccount =
+      DomainAccount(
+        accountRecord.AccountNumber,
+        ZonedDateTime.parse(accountRecord.CreatedDate).withZoneSameInstant(ZoneId.systemDefault()).toLocalDateTime
```

Comment:
zuora returns a zoned date time with UTC+0.  I want to parse it properly, but I wasn't sure whether to convert that to local time, or to use ZonedDateTime.now to do the comparison.  I started to feel like the latter would be better